src/value.rs

```diff
@@ -142,6 +142,7 @@ extern "C" {
   fn v8__Value__BooleanValue(this: *const Value, isolate: *mut Isolate)
     -> bool;
   fn v8__Value__GetHash(this: *const Value) -> int;
+  fn v8__Value__TypeOf(this: *const Value, isolate: *mut Isolate) -> String;
```

Comment:
`*const String` - V8 `String` is a reference not a owned String that v8 provides

Reply:
So my original commit was correct in passing the Isolate as a mutable reference?

Reply:
No - this is correct you need to pass raw pointers between Rust and C++